src/app/people/constants.ts

```diff
@@ -0,0 +1,44 @@
+export const playerList = [
+  {
+    name: 'MOSEUNG',
+    position: 'leader',
+    job: 'Front-End',
+    description: '모승입니다.',
+  },
+  {
+    name: 'Lian',
+    position: 'manager',
+    job: 'Front-End',
+    description: '리안입니다.',
+  },
+  {
+    name: 'Jane',
+    position: 'manager',
+    job: 'Front-End',
+    description: '제인입니다',
+  },
+  {
+    name: 'Dummy',
+    position: 'user',
+    job: 'Front-End',
+    description:
+      '자기 개발은 목표를 설정하고 달성하기 위한 여정입니다. 이 블로그 포스트에서는 일상 생활에 쉽게 통합할 수 있는 5가지 핵심 습관을 소개합니다. 첫 번째는 목표 설정과 시간 관리입니다. 이는 개인적 성취와 전문적 성장을 위한 기초를 마련합니다. 두 번째 습관은 긍정적 사고를 통한 자기 격려입니다. 이는 도전을 극복하고 성공으로 나아가는 데 중요합니다. ',
+  },
+  {
+    name: 'Jane',
+    position: 'manager',
+    job: 'Front-End',
+    description: '제인입니다',
+  },
+  {
+    name: 'Jane',
+    position: 'manager',
+    job: 'Front-End',
+    description: '제인입니다',
+  },
+];
```

Comment:
```suggestion
] as const;
```

여기에 as const를 넣어주고, people/page.tsx에서 타입단언을 없애주는게 더 좋을 것 같아요!!